packages/apollo-env/src/fetch/fetch.ts

```diff
@@ -0,0 +1,100 @@
+import { Agent as HttpAgent } from "http";
+import { Agent as HttpsAgent } from "https";
+
+// declare module "node-fetch" {
```

Comment:
Everything commented out here is intended to be deleted. Left as a TODO for now.